Dockerfile.template

```diff
@@ -7,6 +7,15 @@
 FROM centos:7
 MAINTAINER Crate.IO GmbH office@crate.io
 
+LABEL org.label-schema.schema-version="1.0" \
+    org.label-schema.build-date="AAA" \
+    org.label-schema.name="crate" \
```

Comment:
is it `crate` because of the docker image? or could we name it `CrateDB`?

Reply:
"A human friendly name for the image.", so the former.